src/lib/util.js

```diff
@@ -49,6 +49,17 @@
             return byId( window.location.hash.replace( /^#\/?/, "" ) );
         };
 
+        // `getUrlParamValue` return a given URL parameter value if it exists
+        // `undefined` if it doesn't exist
+        var getUrlParamValue = function( parameter ) {
+            var chunk = window.location.search.split( parameter + "=" )[ 1 ];
+            var value = chunk && chunk.split( "&" )[ 0 ];
+
+            if ( value ) {
```

Comment:
Good point, `chunk`  will be either be a `string` or `undefined`. as you mentioned `0` is a `falsy` value but `"0"` is not  `"0" === true && 0 === false` . So i can't see it feeling in that case.

Reply:
Hmm... Ok, but it's still confusing.

Is this what you're trying to do:

    if ( value === "" or value === undefined ) {
        return value
    }

I'd just prefer it to be more specific so a reader can clearly tell what you wanted to do here, and not need to remember all the possible javascript oddities of how a value could convert to true or false.

Reply:
I can see the readability concerns and i do prefer explicitness over implicitness, but not sure if it's really needed here. Since all i'm trying to do is return `undefined` instead of empty `string` for  consistency sake. if that edge case is ever to occur:

 ```javascript
if ( value !== "") return value
// Otherwise JS will return `undefined`
```

However i tend to be a bad judge of readability 😅, so happy to update it.

Reply:
Resolved with https://github.com/impress/impress.js/pull/764/commits/8f78d8caa2ddd6faf737f1e6c37d0a1ff9efa6e6